repos/system_upgrade/el7toel8/actors/checknfs/actor.py

```diff
@@ -0,0 +1,45 @@
+from leapp.actors import Actor
+from leapp.models import StorageInfo, Inhibitor
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckNfs(Actor):
+    name = "check_nfs"
+    description = "Check if NFS filesystem is in use. If yes, inhibit the upgrade process."
```

Comment:
Please use a docstring now for the description - First line a short summary, empty line and  a body with a bit more detail.